triton/client/client.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Comment for the file `triton/client/1089-134686-0001.json`

Where is it used? Can we remove it?

Reply:
It's generated by "python3 generate_perf_input.py test_wavs/1089-134686-0001.wav <input.json>",  which is used by perf_anyalzer. I would remove it and update the reademe.md.